include/platform/tasklet.h

```diff
@@ -0,0 +1,88 @@
+/*
```

Comment:
You don't want to make an exception for `otSignalTaskletPending` and just put it in `include/openthread-tasklet.h`? If not, I'd choose for `#1` above.